apps/frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -68,31 +69,54 @@ const renderComponent = async () => {
   return result;
 };
 
-it('renders the research output', async () => {
-  mockGetResearchOutput.mockResolvedValue({
-    ...createResearchOutputResponse(),
-    title: 'Proposal title!',
+describe('proposal research output', () => {
+  it('renders a research proposal', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Proposal',
+      team: {
+        displayName: 'Proposal Team',
+        id: '123',
+      },
+      title: 'Proposal title!',
+    });
+    const { getByRole } = await renderComponent();
+    expect(getByRole('heading').textContent).toEqual('Proposal title!');
```

Comment:
If the non-proposal one has a specific assertion that wouldn't work with a proposal (tags), can the proposal one also have a specific assertion that wouldn't work with a non-proposal?